projects/adventure/adv.py

```diff
@@ -22,8 +22,65 @@
 player = Player("Name", world.startingRoom)
 
 # Fill this out
-traversalPath = []
 
+oppositeDirs = {'n': 's', 's': 'n', 'e': 'w', 'w': 'e'}
+traversalPath =[] # fill this out with directions that traverse the whole graph
+graph = {}
+
+def makeQuestions():
+    graph[player.currentRoom.id] = {}
+    for exit in player.currentRoom.getExits():
+        graph[player.currentRoom.id][exit] = '?'
+    # print (graph)
+
+
+def getPossibleDirections():
+    availableExits = []
+    exits = player.currentRoom.getExits()
+    # print('exits', exits)
+    if len(exits) > 0: # makes sure there is at least 1 exit in room
+        for exit in player.currentRoom.getExits():
+            # print('exit', exit)
+            if graph[player.currentRoom.id][exit] == '?':
+                availableExits.append(exit)
+        if len(availableExits) > 0:
+            if len(availableExits) == 1:
+                return availableExits[0]
+            elif len(availableExits) == 2:
+                return availableExits[random.randint(0,1)]
+            elif len(availableExits) == 3:
+                return availableExits[random.randint(0,2)]
+            elif len(availableExits) == 4:
```

Comment:
this could have just been an else: return availableExits[random.randint(0,3)] right?